.github/workflows/ci.yml

```
@@ -12,6 +12,7 @@ jobs:
           - "2.7"
           - "3.0"
           - "3.1"
+          - "3.2.1"
```

```suggestion
          - "3.2"
```
Actions should automatically use the latest 3.2 if we omit the minor.